test/e2e/azure_test.go

```diff
@@ -141,7 +142,7 @@ var _ = Describe("Workload cluster creation", func() {
 
 	if os.Getenv("LOCAL_ONLY") != "true" {
 		Context("Creating a private cluster", func() {
-			It("Creates a public management cluster in the same vnet", func() {
+			It("Creates a public management cluster in a custom vnet", func() {
```

Comment:
this should already be part of https://github.com/kubernetes-sigs/cluster-api-provider-azure/commit/c4064ec43e554d0979eee47bb256fac469257895#diff-93ce65d81f5c1510135f6035d43f678bf9b5962b8e4ee45105e3345a1852efc6R145, rebase error?